microfonia/templates/base.html

```diff
@@ -1,60 +1,86 @@
 {% load static %}
 
-<link rel="stylesheet" href="{% static 'microfonia/template.css' %}" type="text/css" charset="utf-8">
-<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
 
 
-{% block css %} CSS ESPECIFICO AQUI {% endblock %}
 
-<script src="https://use.fontawesome.com/e09838e29b.js"></script>
 
-<style>
-	@font-face {
-  	font-family: 'Raleway';
-  	src: url('{% static 'microfonia/fonts/Raleway-Bold.ttf' %}');
-  	font-weight: normal;
-}
-@font-face {
-  	font-family: 'Raleway';
-  	src: url('{% static 'microfonia/fonts/Raleway-Black.ttf' %}');
-  	font-size: 1em;
-  	font-weight: bold;
-}
-</style>
+<head>
+	<title>Microfonia Produções</title>
+	<meta name="viewport" content = "width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="{% static 'microfonia/template.css' %}" type="text/css" charset="utf-8">
+	<link rel="stylesheet" href="{% static 'microfonia/template-mobile.css' %}" type="text/css" charset="utf-8">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+	<link href="https://fonts.googleapis.com/css?family=Lato:300,700" rel="stylesheet">
 
+	{% block css %}{% endblock %}
 
-<body class="bg-format grey-background">
-	<div class="header">
-		<div class="col-md-2">
-			<img src="{% static 'microfonia/img/thumbnail3.jpg' %}" alt="thumbnail" class="logoheader"/>
+	<script src="https://use.fontawesome.com/e09838e29b.js"></script>
+</head>
+
+<body class="bg-format">
+
+
+
+	<div>
+		<button class="menu-button button-font"><i id='menu-open' class="fa fa-bars" aria-hidden="true"></i><i id='menu-close' class="fa fa-times hide" aria-hidden="true"></i>MENU</button>
```

Comment:
na verdade não precisava ser tag I, podia ser qualquer tag de texto. A tag I é pra voce definir um outro estilo de texto dentro de uma frase, tipo uma palavra em itálico, ou um sublinhado. Como geralmente eu uso esse 'font-awesome' que é uma fonte pra esses simboloszinhos que usa-se bastante na internet, ele muitas vezes vem junto de outro texto com outra formatação, aí por isso boto na I. Nesse caso não fez diferença.